frontend/src/views/MovieSessionDetailsScreen.vue

```diff
@@ -68,7 +68,7 @@ export default {
     async fetchMovieSession (id) {
       try {
         this.loading = true;
-        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}`, {
+        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions/${id}`, {
```

Comment:
```suggestion
        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions/${id}/`, {
```
Make this project one styled, you missed some other endpoints, add `/` at the end.

Reply:
Thanks for the feedback. Fixed all urls used in axios calls.